torch_xla/distributed/xla_dist.py

```diff
@@ -106,6 +106,11 @@ def __init__(self,
 
   def _check_client_mesh_health(
       self, uneven_health_timeout=900, even_health_timeout=1800):
+    uneven_health_timeout = xu.getenv_as(
```

Comment:
These should be coming as parameters.
The API does not look right this way, as you can override a parameter the caller has explicitly set.
You do not know at this point whether it is the default, or a user specified argument.

Reply:
Ah yes. Updated to read it before method call.